docs/fides/docs/tutorial/index.md

```diff
@@ -0,0 +1,87 @@
+# Tutorial Overview
+
+In this tutorial you will learn how to use the Fidesctl open-source developer tools to solve a real-world data privacy problem. The steps closely follow the commits found in [the `ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo).
+
+You will run a local instance of a basic web app to demonstrate the use of Fidesctl as part of a "real" project that uses:
+
+* Flask to run a web server simulating the basic e-commerce application
+* PostgreSQL as the application's database
+* SQLAlchemy to connect to the database
+* [`fidesctl`](https://github.com/ethyca/fides) to declare privacy manifests and evaluate policies
+
+The app itself is simply the [Flask tutorial app](https://flask.palletsprojects.com/en/2.0.x/tutorial/), but modified to simulate an e-commerce marketplace. This helps to highlight some basic examples of data categories that might be stored in a "real" user-facing application.
+
+## Setup Instructions
+
+### System Requirements
+
+Before beginning, ensure you have the following software installed and configured to your liking:
+- Docker (v12+)
+- Python (v3.7+)
+- Make
+
+### Installation
+
+1. Clone [the `ethyca/fidesdemo` repository](https://github.com/ethyca/fidesdemo) to your machine.
+1. Navigate to the repository directory in your command line, and run:
+
+    ```
+    make install
```

Comment:
Hmm, if you do this before checking out the `tutorial-start` tag, this'll do everything before you really get going (run all the docker images, install fidesctl, etc.). And then they'll be running the whole time so you don't get to see the differences when you add new services, etc.

I think for the tutorial, the first step should be to roll back to the first commit and explain why we do that (e.g. "you can follow along by seeing the commit history or try to build it all yourself")